### DISCUSSION TITLE
I can't access the / setup command

### DISCUSSION THREAD

**Discussion by EudesCharlot** [Pryaxis/TShock#2301] [Q&A]
I can't access the / setup command

* TShock version:
* TShock build number (if known):

#### Reproduction steps (if applicable)?

1. Some step
2. Some other step
3. Some bigger step
4. This is the problem

#### Any stack traces or error messages (if known)?

```
PUT SUPER LONG ERROR MESSAGES IN THE TICK MARKS
```

#### What plugins and what versions of those plugins are you running?

If I didn't provide any logs this issue, please close my issue immediately. I'm sorry for the inconvenience.

**Comment by moisterrific** [comment]
Can you provide more info plz? 

What message do you get when you try to use `/setup`? 

Are you using it thru the console or in-game? 

Are you the server host or are you using a hosting service?

**Comment by EudesCharlot** [comment]
before the update when I started the server the console would tell me the code to create an admin account in game etc
now I no longer have the code to put next to / setup I don't know where it was found

**Reply by moisterrific** [reply]
`/setup` is a one time thing, if you've already set it up as owner you will not need to do it again the next time you start TShock. 

Just log in with the account you first registered as owner.
